xray/commands/audit/generic/auditmanager.go

```diff
@@ -238,12 +238,12 @@ func doAudit(params *Params) (results []services.ScanResponse, isMultipleRoot bo
 		}
 		dependencyTrees, e := getTechDependencyTree(params, tech)
 		if e != nil {
-			errorList.WriteString(fmt.Sprintf("'%s' audit failed when building dependency tree:\n%s\n", tech, e.Error()))
+			errorList.WriteString(fmt.Sprintf("'%s' audit failed building a dependency tree:\n%s\n", tech, e.Error()))
 			continue
 		}
 		techResults, e := audit.Audit(dependencyTrees, params.xrayGraphScanParams, params.serverDetails, params.progress, tech)
 		if e != nil {
-			errorList.WriteString(fmt.Sprintf("'%s' audit command failed:\n%s\n", tech, e.Error()))
+			errorList.WriteString(fmt.Sprintf("'%s' audit scan command failed:\n%s\n", tech, e.Error()))
```

Comment:
It is a bit confusing because we also have `jf scan` command. What's wrong with `audit command`?